Use ComWrappers for IErrorInfo (#6743)
dotnet/winforms: src/System.Windows.Forms/tests/UnitTests/WindowsFormsTests/README.md

Supplementary native parts for WinForms unit tests
==================================================

> This project is native DLL which provide unmanaged parts required to testing of WinForms components. Most interesting part of the tests would be COM objects which given to controls.
> This project builds as native DLL, which can be used as reg-free COM host. That allow run tests and do not worry about polluting the developer/build machines.

Reviewer: ```suggestion
This project is native DLL which provide unmanaged parts required for testing WinForms components. Most interesting part of the tests would be COM objects which given to controls.
This project builds as a native DLL, which can be used as a reg-free COM host. That allow running tests without worrying about "polluting" developer/build machines.
```
